bot/tests/conftest.py

```diff
@@ -65,6 +66,27 @@ def is_build_error(self):
     return [MockIssue(i) for i in range(5)]
 
 
+@pytest.fixture
+def mock_coverity_issues():
+    """
+    Build a list of Coverity issues
+    """
+
+    issues = []
+    for i in range(2):
```

Comment:
You can directly do: 

```
return [
  CoverityIssue(0, {...}, "path/to/issue")
  for line in range(2)
]
```
Always use [list comprehension](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) when building a simple structure in a loop